src/Uno.UI/UI/Xaml/Media/Imaging/SvgImageSource.cs

```diff
@@ -56,7 +57,12 @@ private void OnUriSourceChanged(DependencyPropertyChangedEventArgs e)
 		{
 			UnloadImageData();
 		}
-		InitFromUri(e.NewValue as Uri);
+
+		if (e.NewValue is Uri uri)
+		{
+			InitFromUri(uri);
+		}
```

Comment:
I'd just annotate `InitFromUri` parameter as nullable and revert this change.

Reply:
Updated